src/components/logo/Logo.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import style from './Logo.module.scss'
+
+export function Logo(props) {
+    return (
+        <a className={style.logo__ref} href="#">
+            <img className={style.logo__img} src={props.img} alt="logo"/>
```

Comment:
you also need to specify width and height of the image. you can also pass it as props